### ISSUE TITLE
Make tab switching use classes instead of inline CSS

### ISSUE THREAD

**Issue by picasso** [kint-php/kint#361]
I suggest changing the javascript a bit for rich mode.

Now, if the `display` property for tabs is defined in css styles with the `!important` flag, then tabs switching does not work.
You can argue about situations when such a situation arises, but here I got it when embedding the **Kint** output in another container and where there was no way to avoid the use of `!important`.

My proposal is simple and versatile. With its implementation the tabs will switch both in a normal situation and in the presence of the `!important` flag.

Now the script is like this:
**/resources/js/rich.js:125**
```js
for (var i = 0; i < lis.length; i++) {
    if (i === index) {
        lis[i].style.display = 'block';

        if (lis[i].childNodes.length === 1) {
            el = lis[i].childNodes[0].childNodes[0];

            if (el && kintRich.hasClass(el, 'kint-parent')) {
                kintRich.toggle(el, false);
            }
        }
    } else {
        lis[i].style.display = 'none';
    }
}
```

I suggest changing to:
**/resources/js/rich.js:125**
```js
for (var i = 0; i < lis.length; i++) {
    if (i === index) {
        lis[i].style.setProperty('display', 'block', 'important');

        if (lis[i].childNodes.length === 1) {
            el = lis[i].childNodes[0].childNodes[0];

            if (el && kintRich.hasClass(el, 'kint-parent')) {
                kintRich.toggle(el, false);
            }
        }
    } else {
        lis[i].style.setProperty('display', 'none', 'important');
    }
}
```

**Comment by jnvsor** [comment]
> You can argue about situations when such a situation arises, but here I got it when embedding the Kint output in another container and where there was no way to avoid the use of !important.

I find that hard to believe, there's always a way to avoid the use of important. What exactly is the rule that's causing this?

If necessary I would prefer this js to add/remove a class to the tab contents to be shown, instead of setting display inline. That's how it's done everywhere else in rich.

**Comment by picasso** [comment]
I am embedding Kint output in Query Monitor container - this is such a plugin for debugging WordPress theme and plugins. The problem is that all their CSS already consists of `!important`. I understand developers - they are embedded in any page to display debug results and apparently they had no other way to avoid `!important`.

In any case, my every element of what I insert is already under the effect of `!important`. I have nothing to combat this other than specificity plus the new `!important`. It's just a given. And for tabs, visibility is hidden through inline styles. If it were classes, I would override them and everything would be ok. But you can fight inline style only with the help of JS code.

If you're ready to do it with classes, then that should be fine with me.

**Comment by jnvsor** [comment]
Fixed in 26000362f669994863db6ef3bae84faf6abb8049
